operator-pipeline-images/operatorcert/hydra.py

```diff
@@ -28,6 +31,15 @@ def _get_session() -> requests.Session:
 
     session = requests.Session()
     session.auth = (username, password)
+
+    # Exponential retry backoff for a max wait of ~8.5 mins
+    retries = Retry(
```

Comment:
This is used in several places. Can you please extract it into a separate module or function and reuse it everywhere else?

Reply:
Sure, I'll move it into `utils.py`.